src/promptflow/tests/executor/e2etests/test_activate.py

```diff
@@ -44,14 +43,8 @@ def test_bulk_run_activate(self, dev_connections):
     def test_wrong_flow_activate(self, dev_connections):
```

Comment:
Please update the test name as it is no longer the wrong flow. You can add it to `test_flow_run_activate` and move the flow from `wrong_flows` to `flows`.

Reply:
Changed test name to "test_all_nodes_bypassed" and added check in logs whether the warning "The node referenced by output is bypassed" is successfully logged.